ligo/stablecoin/actions.ligo

```diff
@@ -30,16 +118,18 @@ function credit_to
 ; updated_balance := updated_balance + parameter.amount
 ; const updated_ledger : ledger =
     Big_map.update(parameter.to_, Some (updated_balance), store.ledger)
-; const ups : storage = store with record [ ledger = updated_ledger ]
-} with ups
+} with store with record
+    [ ledger = updated_ledger
+    ; total_supply = store.total_supply + parameter.amount
```

Comment:
Why is total_supply is being incremented here?

Reply:
It's used as a slight workaround which helps us not bothering about total supply updates when working with tokens

Reply:
I don't get it. Can you please explain? What issue does this work around solve?

Reply:
Ok, I changed it so total supply can be modified only from `mint` and `burn` entrypoints